package.json

```diff
@@ -21,34 +21,22 @@
   "dependencies": {
     "@mdi/font": "3.3.92",
     "@tokend/js-sdk": "0.3.15",
-    "@tokend/wallet.js": "git+https://github.com/tokend/wallet.js.git",
-    "accounting": "0.4.1",
     "bignumber.js": "8.0.1",
     "clipboard": "2.0.4",
     "d3": "5.7.0",
-    "flatpickr": "4.5.2",
     "husky": "1.3.1",
     "i18next": "13.1.0",
-    "libphonenumber-js": "1.7.4",
     "lodash": "4.17.11",
     "moment": "2.23.0",
-    "node-polyglot": "2.3.0",
     "numeral": "2.0.6",
-    "tokend-js-sdk": "git+https://github.com/tokend/js-sdk.git",
-    "ua-parser-js": "0.7.19",
-    "vee-validate": "2.1.5",
     "vue": "2.5.21",
     "vue-flatpickr-component": "8.1.0",
-    "vue-markdown": "2.2.4",
```

Comment:
please update the branch to the latest master, vue-markdown is being used already